messenger.go

```diff
@@ -264,6 +264,22 @@ func (m *Messenger) Response(to int64) *Response {
 	}
 }
 
+// Send func will send a message to a user without first receiving a message.
+func (m *Messenger) Send(to Recipient, message string, replies []QuickReply) error {
+	response := &Response{
+		token: m.token,
+		to:    to,
+	}
+
+	err := response.TextWithReplies(message, replies)
```

Comment:
The following nine lines can be simplified to the following, because we aren't doing any extra error handing: `return response.TextWithReplies(message, replies)`